kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -362,7 +362,7 @@ window.ToggleNavbarView = BaseView.extend ({
     initialize: function() {
 
         _.bindAll(this);
-        this.listenTo(this.model, "change", this.render);
+        this.listenTo(this.model, "change:client_server_time_diff", this.render);
```

Comment:
I think the main attribute that should toggle is "is_logged_in". The only possible issue I can see for this is if someone somehow was logged in as a student, and then managed to log in as an admin without first logging out, but I think this is an edge case I can live with.